templates/commandline.tmpl

```diff
@@ -52,11 +52,11 @@ $ sudo mv ./kubectl /usr/local/bin/kubectl
             <div class="row">
                 <div class="col s12 right-align"><a href="{{ .HTTPPath }}/kubeconf" class="btn-large waves-effect waves-light blue">Download Kubeconfig</a></div>
                 <div class="col s12">Once kubectl is installed, you may execute the following:</div>
-            </div>       
+            </div>
             <pre>
                <code class="language-bash">
 echo "{{ .ClusterCA }}" \ > ca-{{ .ClusterName }}.pem
-kubectl config set-cluster {{ .ClusterName }} --server={{ .APIServerURL }} --certificate-authority=ca-{{ .ClusterName }}.pem --embed-certs
+kubectl config set-cluster {{ .ClusterName }} --server={{ .APIServerURL }} --certificate-authority-data={{ .ClusterCA | base64enc }}
```

Comment:
Setting `certificate-authority-data` is not a valid option from the `kubectl config set-cluster` command. 

```
Error: unknown flag: --certificate-authority-data
```

Reply:
Oh ok. Thanks.
I am still learning this stuff.
I will switch it back to what it was.